source/common/upstream/cluster_manager_impl.cc

```diff
@@ -143,16 +142,21 @@ void ClusterManagerInitHelper::maybeFinishInitialize() {
   if (!secondary_init_clusters_.empty()) {
     if (!started_secondary_initialize_) {
       ENVOY_LOG(info, "cm init: initializing secondary clusters");
+      const auto target_type_urls =
+          Config::getAllVersionTypeUrls<envoy::config::endpoint::v3::ClusterLoadAssignment>();
```

Comment:
This seems a regression in terms of reducing the amount of version hardcoding; before we could just do `Config::TypeUrl::get().ClusterLoadAssignment()`, now we need to mention v3..